### PR TITLE
adds a one-liner for getting a vector of random smiles for a molecule

### PR DESCRIPTION
**Pull request by greglandrum**
This can be made more efficient later if we want to do that, but this should already be helpful for the deep learners out there.

### CODE REVIEW

**Comment by greglandrum** [comment]
@thegodone : your feedback on this one would be great

File: Code/PgSQL/rdkit/CMakeLists.txt [hunk 1]
```diff
@@ -50,6 +50,7 @@ endif(NOT DEFINED PostgreSQL_CONFIG_DIR)
 if(NOT DEFINED PostgreSQL_CONFIG)
   set(PostgreSQL_CONFIG "${PostgreSQL_CONFIG_DIR}/pg_config")
 endif(NOT DEFINED PostgreSQL_CONFIG)
+set(PostgreSQL_CONFIG /usr/bin/pg_config)
```
Comment (d-b-w) [inline Code/PgSQL/rdkit/CMakeLists.txt]: seems like part of a different PR

Comment (ptosco) [reply Code/PgSQL/rdkit/CMakeLists.txt]: Good catch. It looks like a locally modified file was committed accidentally with the rest of the PR.

Comment (greglandrum) [reply Code/PgSQL/rdkit/CMakeLists.txt]: Right you are. Good catch. I will remove it

File: Code/GraphMol/SmilesParse/SmilesWrite.cpp [hunk 2]
```diff
@@ -572,6 +573,25 @@ std::string MolToCXSmiles(const ROMol &mol, bool doIsomericSmiles,
   return res;
 }
 
+std::vector<std::string> MolToRandomSmilesVect(
+    const ROMol &mol, unsigned int numSmiles, unsigned int randomSeed,
+    bool doIsomericSmiles, bool doKekule, bool allBondsExplicit,
+    bool allHsExplicit) {
+  if (randomSeed > 0) {
```
Comment (bp-kelley) [inline Code/GraphMol/SmilesParse/SmilesWrite.cpp]: Should be != 0, I believe your test case 0xf00d becomes negative.

Comment (bp-kelley) [reply Code/GraphMol/SmilesParse/SmilesWrite.cpp]: Nope, randomSeed is unsigned...

**Comment by bp-kelley** [comment]
The randomSeed is apparently not working.

**Comment by greglandrum** [comment]
> The randomSeed is apparently not working.

Right you are. Apparently std::srand and std::rand are not consistent cross-platform.
I'll switch to std::random.

**Comment by greglandrum** [comment]
@bp-kelley : ok with this now that the RNG problems are cleared up?

**Pull request merged**